### PR TITLE
Blog to announce support for complex-domain linear programs in Convex.jl

### PR DESCRIPTION
**Pull request by Ayush-iitkgp**
In this blog post, I have described the technical aspects of my project and presented a future roadmap for my gsoc project.

### CODE REVIEW

**Comment by ViralBShah** [comment]
Do mention the mentors and others who have helped with the project.

**Comment by Ayush-iitkgp** [comment]
@ViralBShah suggestion incorporated :)

**Comment by ViralBShah** [comment]
Can we also get a mentor to review this post and +1 it? Should be good to go then.

**Comment by Ayush-iitkgp** [comment]
@madeleineudell and @dvij please review my blog post and provide your feedback else +1 it to get the PR merged.

File: blog/_posts/2016-07-26-announcing-support-for-complex-domain-linear-programs-in-Convex.jl.md [hunk 1]
```diff
@@ -0,0 +1,82 @@
...
+    x1 = x.value
+    
+    # Let's now solve by decomposing complex equality constraint into the corresponding real and imaginary part.
+    p2 = minimize(sum(x), real(A)*x == real(b), imag(A)*x==imag(b), x>=0)
+    solve!(p2)
+    x2 = x.value
+    x1==x2 # should return true
+    
+
+    # Let's now consider an example where the complex variable is used
```
Comment (madeleineudell) [inline blog/_posts/2016-07-26-announcing-support-for-complex-domain-linear-programs-in-Convex.jl.md]: Let's now consider an example using a complex variable

File: blog/_posts/2016-07-26-announcing-support-for-complex-domain-linear-programs-in-Convex.jl.md [hunk 2]
```diff
@@ -0,0 +1,82 @@
...
+3. k-th diagonal of a matrix
+4. construct diagonal matrix
+5. transpose and ctranspose
+6. stacking
+7. sum
+8. trace
+9. conv
+10. real and imag
+
+Now, I am working towards implementing the complex-domain second order conic programming. Meanwhile, I invite the Julia community to play around with the complex-domain LPs. The link to the development branch is [here](https://github.com/Ayush-iitkgp/Convex.jl/tree/gsoc2).
```
Comment (madeleineudell) [inline blog/_posts/2016-07-26-announcing-support-for-complex-domain-linear-programs-in-Convex.jl.md]: Now, I am working towards implementing complex-domain second order conic programming. Meanwhile, I invite the Julia community to play around with the complex-domain LPs. The link to the development branch is [here](https://github.com/Ayush-iitkgp/Convex.jl/tree/gsoc2).

**Comment by madeleineudell** [comment]
Looks good to me!

**Comment by Ayush-iitkgp** [comment]
ping @ViralBShah and @MikeInnes

**Comment by MikeInnes** [comment]
Looks good! If you can update the date to today then ping me I'll merge.

**Comment by Ayush-iitkgp** [comment]
@MikeInnes date updated :)

**Comment by ViralBShah** [comment]
Thanks for pinging.  You need to delete the old file.

**Comment by MikeInnes** [comment]
Great, thanks for the post!

**Pull request merged**